administration/src/bp-modules/stores/StoresImportController.tsx

```diff
@@ -29,6 +31,13 @@ const StoresImportController = (): ReactElement => {
   return <StoresImport fields={storeManagement.fields} />
 }
 
+const CenteredSpinner = styled(Spinner)`
+  z-index: 999;
```

Comment:
suggest to use 500 so we could possible put something before in the future.

Reply:
but what should be before an loading spinner f.e ? even its shown over a modal it should be highest index